app/models/project.rb

```diff
@@ -1,3 +1,10 @@
 class Project < ApplicationRecord
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def contestant_count
```

Comment:
Nice job with this method! I agree this logic should be encapsulated in this model.